mappings/net/minecraft/block/Material.mapping

```diff
@@ -3,54 +3,86 @@ CLASS net/minecraft/class_3614 net/minecraft/block/Material
 	FIELD field_15912 breakByHand Z
 	FIELD field_15913 COBWEB Lnet/minecraft/class_3614;
 	FIELD field_15914 STONE Lnet/minecraft/class_3614;
+		COMMENT Hard solid nonmetallic mineral matter of which rock is made, especially as a building material.
 	FIELD field_15915 BUBBLE_COLUMN Lnet/minecraft/class_3614;
-	FIELD field_15916 SAND Lnet/minecraft/class_3614;
+	FIELD field_15916 AGGREGATE Lnet/minecraft/class_3614;
+		COMMENT A material or structure formed from a loosely compacted mass of fragments or particles.
+		COMMENT Previously called SAND
 	FIELD field_15917 SPONGE Lnet/minecraft/class_3614;
 	FIELD field_15918 REDSTONE_LAMP Lnet/minecraft/class_3614;
 	FIELD field_15919 PORTAL Lnet/minecraft/class_3614;
+		COMMENT Material for the various portal blocks.
 	FIELD field_15920 WATER Lnet/minecraft/class_3614;
 	FIELD field_15921 UNUSED_PLANT Lnet/minecraft/class_3614;
+		COMMENT Not in use, but has foliage color.
 	FIELD field_15922 LAVA Lnet/minecraft/class_3614;
 	FIELD field_15923 LEAVES Lnet/minecraft/class_3614;
-	FIELD field_15924 PART Lnet/minecraft/class_3614;
+	FIELD field_15924 SUBBLOCK Lnet/minecraft/class_3614;
+		COMMENT Material for non-full blocks such as redstone components and torches.
+		COMMENT Previously called PART
 	FIELD field_15925 CACTUS Lnet/minecraft/class_3614;
-	FIELD field_15926 SEAGRASS Lnet/minecraft/class_3614;
+	FIELD field_15926 REPLACEABLE_UNDERWATER_PLANT Lnet/minecraft/class_3614;
+		COMMENT Previously called SEAGRASS
 	FIELD field_15927 STRUCTURE_VOID Lnet/minecraft/class_3614;
-	FIELD field_15928 PACKED_ICE Lnet/minecraft/class_3614;
+		COMMENT Material for structure void block.
+	FIELD field_15928 DENSE_ICE Lnet/minecraft/class_3614;
+		COMMENT Material for ice blocks that do not melt. {@see ICE} for meltable ice.
+		COMMENT Previously called PACKED_ICE
 	FIELD field_15929 liquid Z
 	FIELD field_15930 EGG Lnet/minecraft/class_3614;
+		COMMENT Material for egg blocks, such as dragon and turtle eggs.
 	FIELD field_15931 WOOL Lnet/minecraft/class_3614;
+		COMMENT Material for wool and bed blocks.
 	FIELD field_15932 WOOD Lnet/minecraft/class_3614;
+		COMMENT Material for wood logs, and things crafted from them. Includes the nether stems (nether wood logs).
 	FIELD field_15933 PISTON Lnet/minecraft/class_3614;
 	FIELD field_15934 SNOW_BLOCK Lnet/minecraft/class_3614;
+		COMMENT Material for full sized snow blocks.
 	FIELD field_15935 PLANT Lnet/minecraft/class_3614;
-	FIELD field_15936 CLAY Lnet/minecraft/class_3614;
+		COMMENT Material for plants such as flowers and crops
+	FIELD field_15936 MOB_DERIVED Lnet/minecraft/class_3614;
```

Comment:
@dexman545 
> Realistically, clay is made of organic material

No. No it is not.
![image](https://user-images.githubusercontent.com/6429283/77742077-90451e00-701e-11ea-9327-42f8e2abdcfc.png)